android/java/org/chromium/chrome/browser/BraveRewardsHelper.java

```diff
@@ -0,0 +1,396 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser;
```

Comment:
shouldn't we be putting this stuff under org.brave or something or does that cause problems for browser-android-tabs?

Reply:
How it works on Android. Similar classes by functionality are under the same package names. It won't cause problems with old repo, but it will cause a confusion of why do chromium's classes and our have diff package names.

Reply:
it's just distinguishing between brave code and chrome code so we should probably go through at some point in the future to change all brave from chromium -> brave, but fine for now